flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/sql/WindowAggregateTest.scala

```diff
@@ -29,6 +29,27 @@ class WindowAggregateTest extends TableTestBase {
   private val streamUtil: StreamTableTestUtil = streamTestUtil()
   streamUtil.addTable[(Int, String, Long)]("MyTable", 'a, 'b, 'c)
 
+  
+  
+  @Test
+  def testSortProcessingTime() = {
```

Comment:
I would add these tests to a new class `SortTest`. Please add also tests that check that unsupported operations (desc time sort, time field not primary sort field, etc.) throw an exception.